RevenueCatUI/CustomerCenter/Views/NoSubscriptionsCardView.swift

```diff
@@ -127,6 +128,11 @@ private struct BuySubscriptionButtonStyle: ButtonStyle {
 
     func makeBody(configuration: ButtonStyleConfiguration) -> some View {
         configuration.label
+            .foregroundStyle(
+                Color(colorScheme == .light
+                                  ? UIColor.systemBackground
+                                  : UIColor.secondarySystemBackground)
```

Comment:
Should this depend on the configured color?

Reply:
I've re-tested this, and I think the current solution makes more sense.
The text color is not editable in the dashboard, so makes sense to match the background of the container (having a clear color effect)